content/en/docs/_index.md

```diff
@@ -1,16 +1,28 @@
 ---
-title: "Documentation"
+title: "Welcome to Axway AMPLIFY Shared Services documentation"
 linkTitle: "Documentation"
 weight: 20
+no_list: true
 menu:
   main:
     weight: 20
 ---
 
-{{% pageinfo %}}
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
+Axway AMPLIFY Shared Services documentation is available for collaboration using an open source model as part of the Axway Open Documentation Open Beta program.
 
-This section is where the user documentation for your project lives - all the information your users need to understand and successfully use your project.
+## About the Open Beta program
 
-For large documentation sets we recommend adding content under the headings in this section, though if some or all of them don’t apply to your project feel free to remove them or add your own.
\ No newline at end of file
+The Open Beta enables you to suggest edits and create issues for any page of the documentation using a GitHub account. To learn more, see the [contribution guidelines](/docs/contribution_guidelines/).
```

Comment:
This link is broken.  The contrib guidelines is going to be one single file, stored on axway-open-docs, or one version for each microsite? I think one unique file would be better. So in this case might have to specify the full path - https://axway-open-docs.netlify.app/docs/contribution_guidelines/ ??

Reply:
This links is also broken on the right nav bar and I'd like to check this with @winghonlai as I think these links should work without having to put the full path, as the links in the header to blog and community work even though that content is not in this repo but in the common repo also.
I agree all sites should be using a common file.